# this message file is for the human info observed by the robot
# trial number
int64 id
# action of human, 1: looking at robot face, 2: looking at coloured object, 3: looking away
int64 human_action
# human expression, 1: happy, 2:sad, 3: neutral
int64 human_expression
